vendor/github.com/openshift/api/operatoringress/v1/types.go

```diff
@@ -45,6 +45,8 @@ type DNSRecordSpec struct {
 	// +required
 	RecordType DNSRecordType `json:"recordType"`
 	// recordTTL is the record TTL in seconds. If zero, the default is 30.
+	// RecordTTL will not be used in AWS regions Alias targets, but
```

Comment:
The API definitions are vendored from `openshift/api`. For the changes to this file to take affect, you would want to make these same changes in the `openshift/api` github repo (specifically here https://github.com/openshift/api/blob/master/operatoringress/v1/types.go). Then, when we update the vendored version of `openshift/api` in `go.mod`, your changes would propagate into the ingress operator's repo :smile:

Reply:
Thanks Steve.  I noticed this in the CI output yesterday and created [a PR](https://github.com/openshift/api/pull/786) in openshift/api after consulting with Miciah.